train_management/models/train.py

```diff
@@ -74,6 +87,43 @@ def __str__(self):
         return self.label
 
 
+class TrainTimetableTemplate(AbstractDvzoModel):
```

Comment:
Is there a reason this model can't inherit from the TrainTimetable and extend with additional fields?

Reply:
At the time i created it I wasn't aware of that possibility